pkg/security/ebpf/kernel/kernel.go

```diff
@@ -375,6 +376,19 @@ func (k *Version) HaveFentrySupportWithStructArgs() bool {
 	return k.commonFentryCheck("audit_set_loginuid")
 }
 
+// HaveFentryNoDuplicatedWeakSymbols returns whether the kernel supports fentry probes with struct arguments
+func (k *Version) HaveFentryNoDuplicatedWeakSymbols() bool {
+	var symbol string
+	switch runtime.GOARCH {
+	case "amd64":
+		symbol = "__ia32_sys_setregid16"
+	default:
+		return true
```

Comment:
Does ARM don't have duplicate weak symbols?

Reply:
The fix for this issue was merged in the same release as the support for fentry on arm64 so no I don't think it's possible to have this issue on arm64

Happy to add a check anyway if you think it would be better

Reply:
Ok! Thanks for explanation, I was curious :). That's totally fine as this.